src/parser/parsers/heroes.py

```diff
@@ -184,4 +197,4 @@ def _calc_bullet_velocity(self, spline):
                 raise Exception('Bullet speed curve is not constant')
 
         # If constant, return the y
-        return last_y / ENGINE_UNITS_PER_METER
+        return int(round(last_y / ENGINE_UNITS_PER_METER, 0))
```

Comment:
Why is only this one rounded?

Reply:
mistake with conflict merge; just scrapped this